classes/class-wc-rest-connect-shipping-label-controller.php

```diff
@@ -54,7 +54,7 @@ public function post( $request ) {
 			}
 		}
 
-		if ( ! empty( $last_box_id ) ) {
+		if ( ! empty( $last_box_id ) && $last_box_id !== "individual" ) {
```

Comment:
I think we need to check for numeric here, to avoid bugs like this in the future.